rows.go

```diff
@@ -3,19 +3,34 @@ package sqlmw
 import (
 	"context"
 	"database/sql/driver"
+	"reflect"
 )
 
+//go:generate go run ./tools/rows_picker_gen.go -o rows_picker.go
+
 // Compile time validation that our types implement the expected interfaces
 var (
-	_ driver.Rows                           = wrappedRows{}
-	_ driver.RowsColumnTypeDatabaseTypeName // TODO
-	_ driver.RowsColumnTypeLength           // TODO
-	_ driver.RowsColumnTypeNullable         // TODO
-	_ driver.RowsColumnTypePrecisionScale   // TODO
-	_ driver.RowsColumnTypeScanType         // TODO
-	_ driver.RowsNextResultSet              // TODO
+	_ driver.Rows = wrappedRows{}
```

Comment:
optional:
To make it consistent, we should add similiar assignments for all newly added wrappedRows types, alternatively we could also remove the last remaining `_ driver.Rows = wrappedRows{}` assignment.